src/libraries/Common/tests/System/Security/Cryptography/PlatformSupport.cs

```diff
@@ -64,6 +64,45 @@ static bool DetermineAlgorithmFunctional(CngAlgorithm algorithm)
             }
         }
 
+        private static bool CheckIfVbsAvailable()
+        {
+#if !NETFRAMEWORK
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return false;
+            }
+#endif
+
+            CngKey key = null;
+
+            try
+            {
+                const CngKeyCreationOptions RequireVbs = (CngKeyCreationOptions)0x00020000;
+#if !NETFRAMEWORK
+                Assert.Equal(CngKeyCreationOptions.RequireVbs, RequireVbs);
+#endif
+
+                key = CngKey.Create(
+                        CngAlgorithm.ECDsaP256,
+                        $"{nameof(CheckIfVbsAvailable)}{CngAlgorithm.ECDsaP256.Algorithm}Key",
+                    new CngKeyCreationParameters
+                    {
+                        Provider = new CngProvider("Microsoft Software Key Storage Provider"),
+                        KeyCreationOptions = RequireVbs | CngKeyCreationOptions.OverwriteExistingKey,
+                    });
```

Comment:
Indentation looks odd.

```suggestion
                key = CngKey.Create(
                        CngAlgorithm.ECDsaP256,
                        $"{nameof(CheckIfVbsAvailable)}{CngAlgorithm.ECDsaP256.Algorithm}Key",
                        new CngKeyCreationParameters
                        {
                            Provider = new CngProvider("Microsoft Software Key Storage Provider"),
                            KeyCreationOptions = RequireVbs | CngKeyCreationOptions.OverwriteExistingKey,
                        });
```